openedx/core/djangoapps/programs/utils.py

```diff
@@ -508,9 +508,15 @@ def _extend_program(self):
             uuid=self.data['uuid']
         )
         program_instructors = cache.get(cache_key)
+        ecommerce_service = EcommerceService()
         is_learner_eligible_for_one_click_purchase = self.data['is_program_eligible_for_one_click_purchase']
+        skus = []
+        applicable_seat_types = self.data['applicable_seat_types']
 
         for course in self.data['courses']:
+            for seat in course['course_runs'][0]['seats']:
```

Comment:
@vkaracic please remind me, we're fetching the 1st course run because it's the latest course run of the course, right?